package.json

```diff
@@ -110,7 +110,9 @@
     "@types/ws": "^4.0.1",
     "@types/yargs": "^8.0.2",
     "@types/yargs-parser": "^15.0.0",
+    "@typescript-eslint/parser": "^4.7.0",
     "@wardpeet/brfs": "2.1.0-0",
+    "acorn-node": "^1.7.0",
```

Comment:
shouldn't need this explicitly, it's browserify's dep, just need to delete and have it regenerate in `yarn.lock`